travis_script.sh

```diff
@@ -13,14 +13,14 @@ fi
 
 case $TEST_SUITE in
   linters)
-    bundle exec rubocop -Dc .rubocop.yml
+    bundle exec rubocop -DRc .rubocop.yml
```

Comment:
this can also be done (and it is normally done) by enable Rails in the `.rubocop.yml`. Doing it by that we do not need to always give it in the commands, and it is also used to configure the rails cops. You just need to add

```
Rails:
  Enabled: true
```